File: app/src/main/java/com/riders/thelab/data/remote/dto/weather/Weather.java
package com.riders.thelab.data.remote.dto.weather;

import com.squareup.moshi.Json;

import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

@Getter
@Setter
@ToString
public class Weather {
    @Json(name = "id")
    private int id;

    @Json(name = "main")
    private String main;

    @Json(name = "description")
    private String description;

    @Json(name = "icon")
    private String icon;
}

File: app/src/main/java/com/riders/thelab/data/remote/dto/weather/Weather.kt
package com.riders.thelab.data.remote.dto.weather

import com.squareup.moshi.Json

data class Weather(
    @Json(name = "id")
    private var id: Int = 0,

    @Json(name = "main")
    val main: String,

    @Json(name = "description")
    val description: String,

    @Json(name = "icon")
    val icon: String
)
